src/app/location/Location.js

```diff
@@ -246,7 +246,9 @@ define([
                     // this needs to be here since it sets the text on the verify btn
                     // and it needs to wait for $(that.verifyMapBtn).button('loading'); to finish
                     // which is async
-                    that.addLineToMap(inProgressData.path);
+                    if (inProgressData && inProgressData['path']) { // eslint-disable-line dot-notation
```

Comment:
Same comment as in the other PR. No need to use brackets (and related eslint comment) here.

Reply:
switched to dot notation